packages/cactus-test-cmd-api-server/src/test/typescript/integration/plugin-import-with-npm-install.test.ts

```diff
@@ -18,80 +18,78 @@ import {
-  apiServerOptions.cockpitPort = 0;
-  apiServerOptions.grpcPort = 0;
-  apiServerOptions.apiTlsEnabled = false;
-  apiServerOptions.plugins = [
-    {
-      packageName: "@hyperledger/cactus-plugin-keychain-memory",
-      type: PluginImportType.Local,
-      action: PluginImportAction.Install,
-      options: {
-        instanceId: uuidv4(),
-        keychainId: uuidv4(),
-        logLevel,
+    const apiServerOptions = await configService.newExampleConfig();
+    apiServerOptions.pluginManagerOptionsJson = pluginManagerOptionsJson;
+    apiServerOptions.authorizationProtocol = AuthorizationProtocol.NONE;
+    apiServerOptions.configFile = "";
+    apiServerOptions.apiCorsDomainCsv = "*";
+    apiServerOptions.apiPort = 0;
+    apiServerOptions.cockpitPort = 0;
+    apiServerOptions.grpcPort = 0;
+    apiServerOptions.apiTlsEnabled = false;
+    apiServerOptions.plugins = [
+      {
+        packageName: "@hyperledger/cactus-plugin-keychain-memory",
+        type: PluginImportType.Local,
+        action: PluginImportAction.Install,
+        options: {
+          instanceId: uuidv4(),
+          keychainId: uuidv4(),
+          logLevel,
+        },
       },
-    },
-    {
-      packageName: "@hyperledger/cactus-plugin-consortium-manual",
-      type: PluginImportType.Local,
-      action: PluginImportAction.Install,
-      options: {
-        instanceId: uuidv4(),
-        keyPairPem: keyPairPem,
-        consortiumDatabase: db,
+      {
+        packageName: "@hyperledger/cactus-plugin-consortium-manual",
+        type: PluginImportType.Local,
+        action: PluginImportAction.Install,
+        options: {
+          instanceId: uuidv4(),
+          keyPairPem: keyPairPem,
+          consortiumDatabase: db,
+        },
       },
-    },
-  ];
-  const config = await configService.newExampleConfigConvict(apiServerOptions);
+    ];
+    const config = await configService.newExampleConfigConvict(
+      apiServerOptions,
+    );
 
-  const apiServer = new ApiServer({
-    config: config.getProperties(),
+    apiServer = new ApiServer({
+      config: config.getProperties(),
+    });
   });
+  afterAll(() => apiServer.shutdown());
 
-  const startResponse = apiServer.start();
-  await t.doesNotReject(
-    startResponse,
-    "failed to start API server with dynamic plugin imports configured for it...",
-  );
-  t.ok(startResponse, "startResponse truthy OK");
+  test(testcase, async () => {
+    const startResponse = apiServer.start();
+    await expect(startResponse).toBeTruthy();
+    expect(startResponse).toBeTruthy();
 
-  const addressInfoApi = (await startResponse).addressInfoApi;
-  const protocol = apiServerOptions.apiTlsEnabled ? "https" : "http";
-  const { address, port } = addressInfoApi;
-  const apiHost = `${protocol}://${address}:${port}`;
-  t.comment(
-    `Metrics URL: ${apiHost}/api/v1/api-server/get-prometheus-exporter-metrics`,
-  );
-
-  test.onFinish(() => apiServer.shutdown());
+    await startResponse;
```

Comment:
@awadhana And then you can delete this line too if you follow the recommendation from above.

Reply:
@petermetz done :)